src/enums/TwitchKeys.ts

```diff
@@ -0,0 +1,4 @@
+export enum TwitchKeys {
+  "clientId" = "hhxtni83sibk1278lmv6bwye1pnurt",
```

Comment:
Is this needed with the new auth flow?

Reply:
Yes. That is needed in order to call the Twitch API functions.

For example: the [authorization requiest](https://github.com/parithon/vscode-twitch-highlighter/blob/e76697defd64adec336f0afb14a45ba8d5e173a9/src/ttvchat/AuthenticationService.ts#L40).